src/style.css

```
@@ -1 +1,79 @@
-/* Styles go here */
+html:root {
```

Check the docs for this feature, :root is usually used just for the variables. https://developer.mozilla.org/en-US/docs/Web/CSS/:root
```suggestion
:root {
```